src/models/history/tree-monitor.ts

```diff
@@ -291,36 +287,65 @@ export const addTreeMonitor = (tree: Instance<typeof Tree>,  manager: TreeManage
 
     return {
         middlewareDisposer,
-
-        withoutUndo<T>(fn: () => T): T {
-            return skipRecording(fn);
-        },
-        withoutUndoFlow(generatorFn: () => any) {
-            return flow(function* __withoutUndoFlow__() {
-                recordingDisabled++;
-                try {
-                    return yield* generatorFn();
-                } finally {
-                    recordingDisabled--;
-                }
-            });
-        },
     };
 };
 
-function isActionFromManager(call: IActionTrackingMiddleware2Call<CallEnv>) {
+// TODO: add tests using this with:
+// - nested actions: I suspect this will allow any child action to disable the undo
```

Comment:
Were these tests added? In the documentation it sounded like the question about children actions that call `withoutUndo()` is answered, that the call is ignored.

Reply:
Good catch. Yes they were added, so I'll delete this comment.